front/tests/007-pathfinding.spec.ts

```diff
@@ -1,40 +1,53 @@
 import { test } from '@playwright/test';
 import { PlaywrightHomePage } from './pages/home-page-model';
+import { ProjectPage } from './pages/project-page-model';
+import { StudyPage } from './pages/study-page-model';
 import PlaywrightRollingstockModalPage from './pages/rollingstock-modal-model';
 import PlaywrightMap from './pages/map-model';
 import VARIABLES from './assets/operationStudies/test_variables';
 import PlaywrightScenarioPage from './pages/scenario-page-model';
 
 test.describe('Testing pathfinding', () => {
-  test('Test pathfinding: no electrified rolling stock for this path throws error', async ({
-    page,
-  }) => {
-    const playwrightHomePage = new PlaywrightHomePage(page);
-    const scenarioPage = new PlaywrightScenarioPage(page);
+  let playwrightHomePage: PlaywrightHomePage;
+  let projectPage: ProjectPage;
+  let studyPage: StudyPage;
+  let scenarioPage: PlaywrightScenarioPage;
+  let playwrightMap: PlaywrightMap;
+  let isScenarioCreated = false;
```

Comment:
Why is it always false ?

Reply:
It is declared false at the start of the test and becomes true when a scenario is created, thus avoiding duplicates later in the test.